alfalfa_worker/jobs/openstudio/lib/workflow/measures/python/measure.rb

```diff
@@ -71,7 +71,11 @@ def run(ws, runner, user_args)
 
     # modify python plugin search paths
     ws.getObjectsByType('PythonPlugin_SearchPaths'.to_IddObjectType).each do |o|
-      python_paths = ["../.venv/lib/python3.7/site-packages", "/usr/local/lib/python3.7/dist-packages"]
+      python_paths = ["/usr/local/lib/python3.7/dist-packages"]
```

Comment:
looks good. nice work getting this to work because it looks painful to figure out. Maybe a bit more documentation?

Reply:
And the custom build of EP will require this path to be updated.